pkg/api/controller.go

```diff
@@ -253,7 +253,7 @@ func (c *Controller) InitImageStore() error {
 	}
 
 	// Enable extensions if extension config is provided
-	if c.Config.Extensions != nil && c.Config.Extensions.Sync != nil {
```

Comment:
I think we also need to check if sync feature is enabled [here](https://github.com/project-zot/zot/blob/main/pkg/api/routes.go#L1271) and [here](https://github.com/project-zot/zot/blob/main/pkg/api/routes.go#L1283)
Also check with Sebi if changes in pkg/cli/root.go are needed.

Reply:
Indeed, Alexei is right. 

Add a check here
https://github.com/project-zot/zot/blob/main/pkg/cli/root.go#L233

Reply:
Also here: https://github.com/project-zot/zot/blob/main/pkg/extensions/extensions.go#L103